apisix/control/v1.lua

```diff
@@ -269,6 +269,13 @@ local function iter_add_get_routes_info(values, route_id)
         if new_route.value.upstream and new_route.value.upstream.parent then
             new_route.value.upstream.parent = nil
         end
+        -- remove healthcheck info
+        new_route.checker = nil
+        new_route.checker_idx = nil
+        new_route.checker_upstream = nil
+        if new_route.clean_handlers then
+            new_route.clean_handlers = {}
+        end
```

Comment:
why not just also set to nil

Reply:
some existing test cases grep logs and expect this to {}

Reply:
Then you can fix the test case as well.
Checking the logs has no meaning for the control api.

Reply:
done